python/foxglove-sdk/build.rs

```diff
@@ -1,13 +1,12 @@
-// use std::io::{ErrorKind, Read};
-use std::path::Path;
-// use std::process::Command;
+use std::{fs, path::Path, process::Command};
 
+// Deprecated. PB serialization will happen on the Rust side.
```

Comment:
any reason not to delete all this in the same PR?

Reply:
The python examples still use the protobuf types; I plan to remove once those are updated